Formula/x/xaos.rb

```diff
@@ -0,0 +1,28 @@
+class Xaos < Formula
+  desc "Real interactive fractal zoomer"
+  homepage "https://xaos-project.github.io/"
+  url "https://github.com/xaos-project/XaoS/archive/refs/tags/4.3.2-cmake2.tar.gz"
+  sha256 "1abb14b4ccb2cf12d8ff3bac787b1bc4f956bf15419d2fdcdc466ffe3e45ce65"
+  license "GPL-2.0-or-later"
+
+  livecheck do
```

Comment:
Are there often pre-releases?

Reply:
Currently no, three or 4 times a year.

Reply:
It seems an old version of the formula is used, even if I already committed/pushed a new version. I'm starting a new pull request, maybe it will use the new version.